pkg/metrics/machinehealthcheck.go

```diff
@@ -45,3 +56,7 @@ func DeleteMachineHealthCheckNodesCovered(name string) {
 func RegisterMachineHealthCheckNodesCovered(name string, count int) {
 	MachineHealthCheckNodesCovered.With(prometheus.Labels{"name": name}).Set(float64(count))
 }
+
+func RegisterMachineHealthCheckRemediationSuccess(name string) {
+	MachineHealthCheckRemediationSuccessTotal.With(prometheus.Labels{"name": name}).Inc()
+}
```

Comment:
`Register` feels like a weird name here since it is used in the `registry.MustRegister` context as well, would `Update` or `Observe` be better?